### ISSUE TITLE
Implicit subclass constructor loses superclass' parameters

### ISSUE THREAD

**Issue by vladimirvelikov** [haxiomic/dts2hx#117]
Hi and thanks for the wonderful tool that dts2hx is!

I've tried creating apollo-server-express externs and in the result the ApolloServer.hx starts as follows:

```
@:jsRequire("apollo-server-express", "ApolloServer") extern class ApolloServer<ContextFunctionParams> extends apollo_server_core.ApolloServerBase<ContextFunctionParams> {
	function new();
```

notice the parameterless constructor.

Yet, the base class extern looks like this:
```

@:jsRequire("apollo-server-core", "ApolloServerBase") extern class ApolloServerBase<ContextFunctionParams> {
	function new(config:Dynamic);
```

The typescript class ApolloServer doesn't contain a constructor (one is automatically provided). Dts2hx seems to not pick up the superclass contstructor parameters when the sublcass doesn't define it's own constructor.

I've tried adding an explicit constructor in the subclass and it got picked up as expected.

**Comment by haxiomic** [comment]
Thanks for the report @vladimirvelikov, this line looks to be the culprit
https://github.com/haxiomic/dts2hx/blob/4f6a7daea017980937d80733618197de5befc292/src/ConverterContext.hx#L501-L504

I think we should only add a default constructor if the class doesn't extend anything

**Comment by haxiomic** [comment]
Published 0.19.1, thanks for your contribution @vladimirvelikov !
